src/main/java/reactor/netty/resources/ConnectionProvider.java

```diff
@@ -81,7 +90,9 @@ static ConnectionProvider newConnection() {
 	 * {@link Connection}
 	 */
 	static ConnectionProvider elastic(String name) {
-		return elastic(name, null, null);
```

Comment:
Keep this and invoke the builder creation just once in `elastic(String name, @Nullable Duration maxIdleTime, @Nullable Duration maxLifeTime)`